ssas/connection.go

```diff
@@ -31,9 +31,10 @@ func createDB() (*gorm.DB, error) {
 
 	// TODO: Allow connection settings to be configured by env vars
 	// https://jira.cms.gov/browse/BCDA-7109
-	sqlDB.SetMaxOpenConns(40)
+	sqlDB.SetMaxOpenConns(60)
```

Comment:
For reference, I checked production and the database-level max connections is pretty high:

```
bcda=> show max_connections;
 max_connections 
-----------------
 1709
```